mozilla_repo_urls/errors.py

```diff
@@ -8,8 +8,13 @@ def __init__(self, url_string) -> None:
 
 
 class UnsupportedPlatformError(RepoUrlsBaseError):
+    @property
+    def supported_platforms(self):
+        return list(self._supported_platforms)
+
     def __init__(self, url_string, platform, supported_platforms) -> None:
+        self._supported_platforms = supported_platforms
         super().__init__(
-            f"Unsupported platform. Got: {platform}. "
-            f"Expected: {supported_platforms}. URL: {url_string}"
+            f"Unsupported version control host. Got: {platform}. "
+            f"Expected one of: {supported_platforms}. URL: {url_string}"
```

Comment:
Nit: Similarly, we're now passing a list of supported hosts rather than platforms. Let's make that clear here.

```suggestion
            f"Expected one of: {supported_hosts}. URL: {url_string}"
```